modules/dashboard/test/TestPlan

```diff
@@ -37,10 +37,10 @@
     page with the following Selection Filter: Site set to the user's site and Pending Approval set to 'Yes'. The
     Site displayed will be the user's site. Check that you are taken to that page (with the Selection Filter 
     correctly set) when you click on the task. 
-15. Verify that a user with 'Violated Scans' permission has a task with the number of violated scans displayed. This
+15. Verify that a user with 'Violated Scans: View all-sites Violated Scans' permission has a task with the number of violated scans displayed. This
     is the number of entries on the MRI Violated Scans page. The Site displayed will always be 'All'. Check that 
     clicking on the task takes you to the Violated Scans page.
-16. Verify that if a user has the 'Access the files in the document repository' permission, the latest documents to 
+16. Verify that if a user has the 'View and upload files in Document Repository' permission, the latest documents to 
```

Comment:
The dashboard should also show the document repository notifications if the user has the "document_repository_delete" permission